src/screens/Election/ElectionBallot.js

```diff
@@ -58,14 +58,14 @@ class ElectionBallot extends Component {
 		const positionsArray = _.orderBy(positions, iterateesPos, orderPos);
 
 		return (
-			<View style = { page }>
+			<SafeAreaView style = { page }>
 				{ this.renderNavBar() }
 				<View style = { contentStyle }>
 					{ this.showBallot(positionsArray) }
 					{ this.renderCand() }
```

Comment:
Consider renaming all instances of abbreviations of candidate, such as `Cand` or `Can` to `Candidate`, as not many characters are being saved and ends up being confusing.